<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="treeWidgetFiles">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
      <widget class="QTableView" name="tableViewDICOMTags"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>15</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLoad_Directory">
    <property name="title">
     <string>Files...</string>
    </property>
    <addaction name="actionLoad_Directories"/>
    <addaction name="actionLoad_File"/>
   </widget>
   <addaction name="menuLoad_Directory"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Directories">
   <property name="text">
    <string>Load Directories...</string>
   </property>
  </action>
  <action name="actionLoad_File">
   <property name="text">
    <string>Load File...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
